Google Translate API still uses snekfetch

Google Translate API still requires `snekfetch`, causing this error:

```
Error: Cannot find module 'snekfetch'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (C:\Users\Pedro\WebstormProjects\switchblade\src\apis\GoogleTranslate.js:2:19)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Promise.all.files.map (C:\Users\Pedro\WebstormProjects\switchblade\src\utils\FileUtils.js:13:28)
    at Array.map (<anonymous>)
    at Function.requireDirectory (C:\Users\Pedro\WebstormProjects\switchblade\src\utils\FileUtils.js:9:30)
```

--- a/src/apis/GoogleTranslate.js
+++ b/src/apis/GoogleTranslate.js
@@ -1,5 +1,5 @@
 const { APIWrapper } = require('../')
-const snekfetch = require('snekfetch')
+const fetch = require('node-fetch')
 
 module.exports = class GoogleTranslate extends APIWrapper {
   constructor () {
@@ -21,7 +21,10 @@ module.exports = class GoogleTranslate extends APIWrapper {
       tl: to,
       q: text
     }
-    const res = await snekfetch.get('https://translate.googleapis.com/translate_a/single?client=gtx&dt=t').query(params).then(r => r.body)
+
+    const URLqueryParams = new URLSearchParams(params)
+    const res = await fetch('https://translate.googleapis.com/translate_a/single?client=gtx&dt=t' + `&${URLqueryParams.toString()}`)
+      .then(res => res.json())
 
     return {
       translated: res[0][0][0],

--- a/src/commands/utility/translate.js
+++ b/src/commands/utility/translate.js
@@ -5,6 +5,7 @@ module.exports = class TranslateCommand extends Command {
     super(client, {
       name: 'translate',
       aliases: ['translator'],
+      requirements: { apis: ['gtranslate'] },
       parameters: [
         {
           type: 'string'

